### PR TITLE
docs: tidy up security.txt and security.md file

### LINKED ISSUE #1408: Mismatch in content of  `SECURITY.txt` in Units and e.g. NGINX Agents / NJS Repositories
Good Friday Unit-Team :) 

feels great to be back here on GitHub with you. While researching I have noticed that there is a difference between the `SECURITY.txt` in this repository and the `SECURTIY.md` in njs and agents repository. As the last two are updated more recently, maybe there is value in using the same Policy Document. If thats accurate, I am MORE than happy to come up with an PR.

PS: Great to see 1.33 on the Horizon AND the Wasmtime Version Bump <3

### PR DESCRIPTION
**Pull request by javorszky**
After a conversation with security folks in other NGINX projects, the recommendation was to completely remove the security.txt file and adopt the security.md file in the [template repository](https://github.com/nginxinc/template-repository).

Closes #1408

### CODE REVIEW

**Comment by thresheek** [comment]
This also needs a followup on the docs repo, since this file is deployed to the website: https://github.com/nginx/unit-docs/blob/main/Makefile#L58

**Review by ac000 — CHANGES_REQUESTED** [review]
```
docs: add SECURITY.md

All new NGINX projects are created from the [template repository](https://github.com/nginxinc/template-repository)
which has a SECURITY.md file in it. This PR adopts the file.

Signed-off-by: Gabor Javorszky <g.javorszky@f5.com>
```

Watch the line wrapping, I would re-work to be something like

```
docs: add SECURITY.md

All new NGINX projects are created from the template repository, which
has a SECURITY.md file in it. This adopts the file.

Link: <https://github.com/nginxinc/template-repository>
Signed-off-by: Gabor Javorszky <g.javorszky@f5.com>
```

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+We advise users to run or update to the most recent release of this project. Older versions of this project may not have all enhancements and/or bug fixes applied to them.
+
+## Reporting a Vulnerability
+
+The F5 Security Incident Response Team (F5 SIRT) has an email alias that makes it easy to report potential security vulnerabilities:
+
+- If you’re an F5 customer with an active support contract, please contact [F5 Technical Support](https://www.f5.com/services/support).
+- If you aren’t an F5 customer, please report any potential or current instances of security vulnerabilities with any F5 product to the F5 Security Incident Response Team at <F5SIRT@f5.com>.
+
+For more information please read the F5 SIRT vulnerability reporting guidelines available at [https://www.f5.com/services/support/report-a-vulnerability](https://www.f5.com/services/support/report-a-vulnerability).
```
[the remark this answers is not in the document]
Comment (javorszky) [reply SECURITY.md]: wrapped. Though need to point out that the file was an exact copy of the one in the template repository, so it's expected that any new repositories will also not have the file contents wrapped.

**Comment by ac000** [comment]
- Rebase with master
- Commit message tweaks

```
$ git range-diff 54b7dce0...cc863e15
-:  -------- > 1:  63148a31 tools/unitctl: whitespace fixes
-:  -------- > 2:  5e8a6893 tools/unitctl: rename app -> apps, fix readme
1:  d80e3be6 = 3:  f2e05bc7 docs: remove security.txt file
2:  54b7dce0 ! 4:  cc863e15 docs: add SECURITY.md
    @@ Metadata
      ## Commit message ##
         docs: add SECURITY.md
     
    -    All new NGINX projects are created from the [template repository](https://github.com/nginxinc/template-repository)
    -    which has a SECURITY.md file in it. This PR adopts the file content.
    +    All new NGINX projects are created from the template repository which
    +    has a SECURITY.md file in it. This adopts the file.
     
    +    NOTE; We wrap the file around the 76-78 character mark for consistency
    +    and readability.
    +
    +    Link: <https://github.com/nginxinc/template-repository>
    +    Closes: https://github.com/nginx/unit/issues/1408
         Signed-off-by: Gabor Javorszky <g.javorszky@f5.com>
    +    [ Tweaked commit message - Andrew ]
    +    Signed-off-by: Andrew Clayton <a.clayton@nginx.com>
     
      ## SECURITY.md (new) ##
     @@
```

**Review by ac000 — APPROVED** [review]

**Pull request merged**